packages/components/src/design-language/index.ts

```diff
@@ -0,0 +1,3 @@
+import designLanguage from '@codesandbox/common/lib/design-language/theme';
+
+export default designLanguage;
```

Comment:
Any specific reason you just re-export `@codesandbox/common/lib/design-language/theme`? 🤔

Reply:
Yep

We replace that file during the build step to "bundle in" the design language. It's a quick way of creating a bundle that can be published to npm without publishing the latest version of common. 

A nicer way of doing this would be to use rollup to resolve internal dependencies.

Reply:
Seems like a good trade-off then indeed 🙂